src/ApiGenerator/RestSpecification/_Patches/security.clear_api_key_cache.patch.json

```diff
@@ -0,0 +1,7 @@
+{
+  "security.clear_api_key_cache": {
+    "url": {
+      "paths": ["/_security/api_key/{ids}/_clear_cache", "/_security/api_key/*/_clear_cache"]
```

Comment:
```suggestion
      "paths": ["/_security/api_key/{ids}/_clear_cache", "/_security/api_key/_clear_cache"]
```

Reply:
If this route is not available, for now patching it with `*` is fine on our end. Upstream this should not be the long term fix.

Reply:
Yeah, that route is not available (yet) so will leave `*` for the patch for the time being.